sdk/hard-forks/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "solana-hard-forks"
+description = "The list of slot boundaries at which a hard fork should occur."
+documentation = "https://docs.rs/solana-hard-forks"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+byteorder = { workspace = true }
+serde = { workspace = true, optional = true }
+serde_derive = { workspace = true, optional = true }
+solana-frozen-abi = { workspace = true, optional = true }
+solana-frozen-abi-macro = { workspace = true, optional = true }
```

CI is failing because you also need to enable the `frozen-abi` feature on these
```suggestion
solana-frozen-abi = { workspace = true, optional = true, features = ["frozen-abi"] }
solana-frozen-abi-macro = { workspace = true, optional = true, features = ["frozen-abi"] }
```